dbConfig.mjs

```diff
@@ -0,0 +1,13 @@
+import mysql from "mysql2";
```

Comment:
We will use postgres instead of mysql. Use this package https://www.npmjs.com/package/pg